_zigbee/Nabu_Casa_HASKYCNCT.md

```diff
@@ -0,0 +1,22 @@
+---
+date_added: 2023-07-10
+model: HASKYCNCT
+vendor: Nabu Casa
+title: Home Assistant SkyConnect
+category: coordinator
+supports: coordinator
+zigbeemodel: 
+compatible: [z2m, zha]
+mlink: https://www.home-assistant.io/skyconnect
+link: 
+---
+
+The Home Assistant SkyConnect is a USB Zigbee coordinator from the founders of Home Assistant. It comes with a preinstalled Zigbee coordinator firmware (EmberZNet). Home Assistant SkyConnect can also be flashed with ZigBee and Thread multiprotocol firmware, or your own firmware.
```

Comment:
```suggestion
The Home Assistant SkyConnect is a USB Zigbee coordinator from the founders of Home Assistant. It comes with a preinstalled Zigbee coordinator firmware (EmberZNet). Home Assistant SkyConnect can also be flashed with a ZigBee and Thread multiprotocol firmware, or your own firmware.
```

Reply:
Thank you! Good point. Removed that information. Any other feedback?